libhb/deccc608sub.h

```diff
@@ -70,6 +70,7 @@ struct eia608
     int ssa_counter; // Number of subs currently written
     int screenfuls_counter; // Number of meaningful screenfuls written
     int64_t current_visible_start_ms; // At what time did the current visible buffer became so?
+    int64_t current_visible_scr_sequence; // At what time did the current visible buffer became so?
```

Comment:
It actually was intended, though maybe I can improve the wording.   I'll change it to say SCR instead of time.

"Time" is now defined by the combination of start and scr_sequence (until a buffer reaches sync.c).  A change in the scr_sequence number indicates a discontinuity in "start" times.